### PR TITLE
feat(ci): add Makefile and .travis.yml

### PR DESCRIPTION
**Pull request by vdice**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	${DEV_ENV_CMD} glide install
+
+build:
+	${DEV_ENV_PREFIX} -e CGO_ENABLED=0 ${DEV_ENV_IMAGE} go build
+
+test:
+	${DEV_ENV_CMD} sh -c 'go test $$(glide nv)'
+
+build-cli-cross:
+	${DEV_ENV_CMD} gox -output="cli/bin/${SHORT_NAME}-{{.OS}}-{{.Arch}}"
```
Comment (bacongobbler) [inline Makefile]: is there a reason we can't shorten this down to `bin/`?

**Comment by arschles** [comment]
I'm good with this, aside from @bacongobbler's [question](https://github.com/deis/deisrel/pull/19#discussion_r62390421)

File: Makefile (hunk 1 above)
Comment (vdice) [inline Makefile]: good call; will amend

Comment (vdice) [inline Makefile]: @bacongobbler amended

**Pull request merged**